server_src/tasks/printer-websocket.task.js

```diff
@@ -16,24 +22,26 @@ class PrinterWebsocketTask {
 
   #logger = new Logger("Printer-Websocket-Task");
 
-  #errorMaxThrows = 5;
-  #errorModulus = 10; // After max throws, log it every 10 failures
+  #errorMaxThrows = 3;
+  #errorModulus = 50; // After max throws, log it every x failures
```

Comment:
Explanation: after 3 errors the logs are silenced. After each 50 retries a log is done. This ends up as ~360 logs per day at most.